src/model/src/Model.cpp

```diff
@@ -703,6 +703,26 @@ LinkIndex Model::getFrameLink(const FrameIndex frameIndex) const
     return LINK_INVALID_INDEX;
 }
 
+bool Model::getLinkAdditionalFrames(const LinkIndex lnkIndex, std::vector<FrameIndex>& frameIndices) const
+{
+    if (!isValidLinkIndex(lnkIndex)) {
+        std::stringstream ss;
+        ss << "LinkIndex " << lnkIndex << " is not valid, should be between 0 and " << this->getNrOfLinks()-1;
+        reportError("Model", "getLinkAdditionalFrames", ss.str().c_str());
+        return false;
+    }
+
+    frameIndices.resize(0);
+    for (FrameIndex frameIndex=this->getNrOfLinks(); frameIndex < this->getNrOfFrames(); frameIndex++) {
```

Comment:
For quicker understanding and readabilityof code, it would be great to document the relationship between the link serialization and the additional frames serialization, also here.

Reply:
I see it is documented here, https://github.com/robotology/idyntree/blob/ed3e020ee38a2b65c08ac5e48454a24f2944197c/src/model/include/iDynTree/Model/Model.h#L63
In that case, we can give a pointer to this documentation here.

Reply:
Fixed in https://github.com/robotology/idyntree/pull/557/commits/81e7f7ab0890d980b6655fe6268b8d692c31f87b .